pkg/geo/geographiclib/geographiclib.go

```diff
@@ -109,3 +114,22 @@ func (s *Spheroid) AreaAndPerimeter(points []s2.Point) (area float64, perimeter
 	)
 	return float64(areaDouble), float64(perimeterDouble)
 }
+
+// Project returns computes the location of the projected point.
+//
+// Using the direct geodesic problem from GeographicLib (Karney 2013).
+func (s *Spheroid) Project(point s2.LatLng, distance float64, azimuth s1.Angle) s2.LatLng {
+	var lat, lng C.double
+
+	C.geod_direct(
+		&s.cRepr,
+		C.double(point.Lat.Degrees()),
+		C.double(point.Lng.Degrees()),
+		C.double(azimuth*180.0/math.Pi),
+		C.double(distance),
+		&lat,
+		&lng,
+		nil)
```

Comment:
nit: i prefer trailing commas and one arg per line, i.e.

```
  nil,
)
```

easier to git blame and less merge conflict-y.

Reply:
Done.